src/Core/Components/Radio/FluentRadio.razor.cs

```diff
@@ -0,0 +1,81 @@
+// ------------------------------------------------------------------------
+// MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------
+
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.AspNetCore.Components;
+using Microsoft.FluentUI.AspNetCore.Components.Extensions;
+using Microsoft.FluentUI.AspNetCore.Components.Utilities;
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+/// <summary>
+/// A Fluent Radio button component.
+/// </summary>
+/// <typeparam name="TValue">The type for the value of the radio button</typeparam>
+public partial class FluentRadio<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)] TValue> : FluentInputBase<TValue>, IFluentField
+{
+    bool _trueValueToggle;
+
+    internal FluentRadioContext? Context { get; private set; }
+
+    /// <summary />
+    public FluentRadio()
+    {
+        Id = Identifier.NewId();
```

Comment:
Why do we force to have an ID? Not working without that?

Reply:
I assumed it was needed for the FluentField. But it isn't, so I've removed the constructor code.